test/optim/test_optim.py

```diff
@@ -1715,8 +1666,7 @@ def test_rmsprop(self):
             self._test_complex_optimizer(
                 lambda param: RMSprop([param], maximize=True, foreach=foreach)
             )
-            with self.assertRaisesRegex(ValueError, "Invalid momentum value: -1.0"):
-                RMSprop(None, lr=1e-2, momentum=-1.0, foreach=foreach)
```

Comment:
This is one place where coverage "decreased", since I no longer test this error case for BOTH foreach T and F. Which I believe is unnecessary anyway.

Reply:
maybe this is unnecessary, but it's easy enough to do this:
```python
class TestOptimRenewed:
    ...
    @onlyCPU
    @optims([optim for optim in optim_db if optim.optim_error_inputs_func is not None])
    @parametrize("foreach", [False, True])
    def test_errors(self, device, dtype, optim_info, foreach):
        ...
```

Reply:
Yea, thanks for the code pointer. I don't believe it is necessary though.